llvm/test/CodeGen/AMDGPU/conditional-mem-no-cbranch-execz.ll

```diff
@@ -0,0 +1,66 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx942 -mattr=+sramecc,-xnack < %s | FileCheck %s
```

Comment:
Shouldn't need the target features

Reply:
Removed them in 345269b1523b1c2af48dc3798519031bf30d218d.